flask_appbuilder/translations/tr/LC_MESSAGES/messages.po

```diff
@@ -123,7 +124,7 @@ msgstr "Silinen Satır"
 #: build/lib/flask_appbuilder/models/base.py:30
 #: flask_appbuilder/models/base.py:30
 msgid "Associated data exists, please delete them first"
-msgstr "İlgili veriler mevcut, lütfen önce bunları silin"
+msgstr "Bağlantılı veriler mevcut, lütfen önce bunları silin"
```

Comment:
Burada "ilişkili" demek daha doğru olmaz mı?
```suggestion
msgstr "İlişkili veriler mevcut, lütfen önce bunları silin"
```

Reply:
Thanks for your suggestion, I committed.